src/gatling/scala/uk/gov/hmcts/reform/featuretoggle/actions/Delete.scala

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.featuretoggle.actions
+
+import com.typesafe.config.{Config, ConfigFactory}
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ChainBuilder
+import io.gatling.http.Predef._
+import uk.gov.hmcts.reform.featuretoggle.actions.util.ToggleIdStore
+
+object Delete {
+
+  private val config: Config = ConfigFactory.load()
+
+  val delete: ChainBuilder =
+    feed(ToggleIdStore.getAll().map(id => Map("deleteId" -> id)))
```

Comment:
key has to different than the one used in 'create' step. Otherwise gatling says that the feeder is empty (reached the last element)